requirements.txt

```diff
@@ -4,7 +4,7 @@ sounddevice~=0.4.1
 keyboard~=0.13.5
 mouse~=0.7.1
 SoundFile~=0.10.3.post1
-numpy~=1.19.5
+numpy~=1.17.3
 ffmpeg~=1.4
 tensorflow~=2.3.2
-#tensorflow-gpu~=2.3.2
+matplotlib~=3.3.4
```

Comment:
Can we also switch to Pipenv with this PR? please please

Reply:
Would we all **have** to use Pipenv, or will there still be a `requirements.txt`?
Anyways, I don't mind, but you will do it. And I recommend for that to be in a separate PR

Reply:
Yeah, Pipenv is a replacement for `requirements.txt`. Alright then, I'll make a PR, see what people think